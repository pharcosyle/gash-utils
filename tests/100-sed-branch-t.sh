seq 3 | sed -e '
2 s/\(.*\)/\1/
: repeat
a foo
t repeat'
