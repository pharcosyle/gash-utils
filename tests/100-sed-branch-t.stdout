1
foo
2
foo
foo
3
foo
